### PR TITLE
Topic recognition - DCGAN_Ryan

### PR DESCRIPTION
**Pull request by Ryan-RuixiangMa**
create a pull-request

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Commit messages should be more specific
- Code valid for Hard Difficulty
- Results are good

**Comment by rexsw** [comment]
Code looks good for hard difficulty

For next time;

Your pull request should have a description of what it is and why it should be merged 

Comments such as 
     # Create the models
     generator = make_generator_model()
Add nothing to the readability of the code and should be avoided. But overall comments are good. 

Make sure you remove unused imports

File: recognition/DCGAN_Ryan/main.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
+import tensorflow as tf
```
Comment (shakes76) [inline recognition/DCGAN_Ryan/main.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme OK
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Pull request merged**